data/scripts/cable_club.inc

```diff
@@ -1257,7 +1257,14 @@ EventScript_CloseMossdeepGameCornerBarrier::
 	return
 
 CableClub_OnResume:
+.if OW_UNION_DISABLE_CHECK == FALSE
+.if OW_UNION_CONNECTED_ONLY_CHECK == TRUE
+	specialvar VAR_0x8001, IsWirelessAdapterConnected
+	goto_if_eq VAR_0x8001, FALSE, CableClub_OnResume_End
+.endif
 	special InitUnionRoom
+.endif
```

Comment:
```suggestion
.endif @ OW_UNION_CONNECTED_ONLY_CHECK == TRUE
	special InitUnionRoom
.endif @ OW_UNION_DISABLE_CHECK == FALSE
```
Just for clarity because of stacking .ifs